build.gradle

```diff
@@ -6,18 +6,18 @@ buildscript {
     'sourceCompatibility': JavaVersion.VERSION_1_7,
     'targetCompatibility': JavaVersion.VERSION_1_7,
     'okhttp': '3.10.0',
-    'supportLibrary': '28.0.0-alpha3'
+    'supportLibrary': '1.0.0-rc01'
```

Comment:
We can't merge until 1.0 final.

Reply:
totally curious: why not?

Reply:
I don't want to start inflicting this on people forcefully until it's stable. Granted, we aren't going to have a release right away or anything.

Anyway there's going to be an RC2 any second now and with any luck 1.0 final will be a week after that.

Reply:
cool, i was just curious if it was a timebomb thing again or something weird.

Reply:
@JakeWharton any news on the RC2 and/or 1.0 final release?